values/istio-gateway/istio-egressgateway.yaml.gotmpl

```diff
@@ -0,0 +1,24 @@
+{{- $v := .Values }}
+{{- $i := $v.apps.istio }}
+
+name: istio-egressgateway{{- if $i.canaryRevision }}-{{ $i.canaryRevision }}{{ end }}
+revision: {{ $i.canaryRevision | quote }}
+
+{{- with .Values.otomi | get "globalPullSecret" nil }}
+imagePullSecrets:
+  - name: otomi-pullsecret-global
+{{- end }}
+
+labels:
+  istio: egressgateway{{- if $i.canaryRevision }}-{{ $i.canaryRevision }}{{ end }}-public
```

Comment:
I recommend to align on label and resource namings in dervied.gotmpl file

Reply:
Done.